bears/haml/HAMLLintBear.py

```diff
@@ -0,0 +1,396 @@
+                    'style': id_names_style,
+                },
+                'ImplicitDiv': {
+                    'enabled': implicit_div,
+                },
+                'Indentation': {
+                    'enabled': indentation,
+                    'character': indentation_character,
+                    'width': indentation_width,
+                },
+                'InlineStyles': {
+                    'enabled': inline_styles,
+                },
+                'InstanceVariables': {
+                    'enabled': instance_variables,
+                    'file_types': instance_variables_file_types,
+                    'matchers': {
+                        'all': instance_variables_matchers_all,
+                        'partials': instance_variables_matchers_partials,
+                    },
+                },
+                'LeadingCommentSpace': {
+                    'enabled': leading_comment_space,
+                },
+                'LineLength': {
+                    'enabled': line_length,
+                    'max': max_line_length,
+                },
+                'MultilinePipe': {
+                    'enabled': multiline_pipe,
+                },
+                'MultilineScript': {
+                    'enabled': multiline_script,
+                },
+                'ObjectReferenceAttributes': {
+                    'enabled': object_reference_attributes,
+                },
+                'RepeatedId': {
+                    'enabled': repeat_id,
+                    'severity': repeat_id_severity,
+                },
+                'RuboCop': {
+                    'enabled': rubo_cop,
+                    'ignored_cops': rubo_cop_ignored_cops,
+                },
+                'RubyComments': {
+                    'enabled': ruby_comments,
+                },
+                'SpaceBeforeScript': {
+                    'enabled': space_before_script,
+                },
+                'SpaceInsideHashAttributes': {
+                    'enabled': space_inside_hash_attributes,
+                    'style': space_inside_hash_attributes_style,
+                },
+                'TagName': {
+                    'enabled': tag_name,
+                },
+                'TrailingWhitespace': {
+                    'enabled': trailing_whitespace,
+                },
+                'UnnecessaryInterpolation': {
+                    'enabled': unnecessary_interpolation,
+                },
+                'UnnecessaryStringOutput': {
+                    'enabled': unnecessary_string_output,
+                },
+                'ViewLength': {
+                    'enabled': view_length,
+                    'max': max_view_length,
+                },
+            },
+        }
+
+        return yaml.dump(hamllint_config, default_flow_style=False)
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         hamllint_config: str=''):
```

Comment:
Please also put parameter into `generate_config`, and put the config generation into an if:
```
if hamlling_config:
    return None
else:
    # code from before
```
This will increase performance by avoiding unnecessary config file generations. As if a user has a custom hammlint config, we don't need to generate one ;)